test/common/config.js

```diff
@@ -69,7 +69,7 @@ export default function () {
         // These are *very* important to have them pointing to staging
         // '--telemetry.url=https://telemetry-staging.opensearch.org/xpack/v2/send',
         // '--telemetry.optInStatusUrl=https://telemetry-staging.opensearch.org/opt_in_status/v2/send',
-        `--server.maxPayloadBytes=1679958`,
```

Comment:
this is required because this value was generated prior to us changing to opensearch-dashboards. So in large payloads tests, the size got even larger so this value was generated getting the value of the payload size and setting it as the limit.

Reply:
nice